env.cpu.mac.yml

```diff
@@ -9,7 +9,5 @@ dependencies:
     - torch-scatter
     - torch-sparse
     - torch-spline-conv
-    - openvino==2021.2
-    # Use after merge of https://github.com/openvinotoolkit/openvino/pull/3123
-    # - git+git://github.com/openvinotoolkit/openvino.git#subdirectory=model-optimizer
-    - git+git://github.com/dkurt/openvino.git@mo_pip#subdirectory=model-optimizer
+    - openvino==2021.3
```

Comment:
seems like there is no Python 3.8 for Linux :(
Let's use 2021.2 (2021.3 MO is ok)